src/report/controller/report-calculation.controller.ts

```diff
@@ -71,7 +74,7 @@ export class ReportCalculationController {
           }
 
           return this.reportStorage
-            .fetchReport(new Reference(calculation.report.id), token)
+            .fetchReport(new Reference(calculation.report.id))
```

Comment:
This one is special, I think:

We want to control access to individual ReportConfig (and implied by that, to the results of that report) on a user-basis with our normal Config:Permission rules. (e.g. some user may have access to ReportConfig:1 but not to ReportConfig:2).

Possibly something for a follow up issue, but a mostly confirmed requirement.

Reply:
Yes, this need some filter logic that was currently handled by the replication-backend. Will be another issue.